draft-barnes-acme.md

```diff
@@ -133,9 +133,6 @@ Subject Public Key:
 Account Key Pair:
 : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
 
-Recovery Token:
```

Comment:
As you suggest, the limit here is the size of the token the user is willing to remember, and that can just as easily be used to generate an asymmetric key as a symmetric key.

Technical note: There are a bunch of things you could imagine doing, like, say an S/Key style hash chain, that would have better MITM resistance than you suggest above.